projects/graph/social/social.py

```diff
@@ -45,11 +45,23 @@ def populateGraph(self, numUsers, avgFriendships):
         self.users = {}
         self.friendships = {}
         # !!!! IMPLEMENT ME
-
+        from itertools import combinations
+        import random
         # Add users
+        for i in range(numUsers):
+            self.addUser(f'User{i}')
 
         # Create friendships
 
+        allPossibleFrienships = list(combinations(range(1, numUsers+1), 2))
+        print(allPossibleFrienships)
+        random.shuffle(allPossibleFrienships)
```

Comment:
awesome job implementing your shuffle, I know you've been working really hard bryan, you're doing great!